app/controllers/transactions_controller.rb

```diff
@@ -87,7 +92,9 @@ def transaction_params
       :amount,
       :cutoff_date,
       :due_date,
-      :actualized_at
+      :actualized_at,
+      :created_by,
+      :updated_by
```

Comment:
Params either comes from data entered by user on form or url parameters.  So where are `:created_by` and `:updated_by` coming from?